src/main/kotlin/fi/metatavu/metaform/server/controllers/MetaformController.kt

```diff
@@ -366,4 +373,13 @@ class MetaformController {
 
         return true
     }
+
+    /**
+     * Marks a form as deleted so that a scheduled job deletes it later
+     *
+     * @param metaform
+     */
+    fun setMetaformDeleted(metaform: Metaform) {
+        metaformDAO.setMetaformDeleted(metaform)
+    }
```

Comment:
usually method's that update fields are called updateXXX:

```suggestion
    /**
     * Marks a form as deleted so that a scheduled job deletes it later
     *
     * @param metaform metaform to be deleted
     */
    fun updateMetaformDeleted(metaform: Metaform) {
        metaformDAO.updateMetaformDeleted(metaform)
    }
```